client/src/components/river/index.tsx

```diff
@@ -109,17 +113,26 @@ const FhirRiverView = (): React.ReactElement => {
 
   const handleClickCreateBatch = async (): Promise<void> => {
     setRunning(true);
+    const runOnce = selectedFrequency === "run once"
```

Comment:
naming confused me at first, I suggest 
```suggestion
    const isRecurring = selectedFrequency !== "run once"
```

Reply:
I think it will make the rest of this function clearer